doc/sphinx/src/sphinx-doc.rst

```diff
@@ -24,29 +24,27 @@ Building documentation locally
 
 While you can rely on the CI to build the documentation associated with your
 branch, you can also very easily build sphinx documentation locally through
-python. These instructions also _do not_ require admin access and are usable
+python. These instructions also *do not* require admin access and are usable
 with shared machines or python distributions.
 
 First, ensure that you are running a modern version of python (i.e. python 3 of
 some flavor)
 
-.. code-block::
-   :language:bash
+.. code-block:: bash
+
```

Comment:
This is on me thanks for the fix. I had fixed this in my metadata PR. But too late for that now. :)